devtools/create-gossipstore.c

```diff
@@ -4,24 +4,44 @@
 #include <ccan/read_write_all/read_write_all.h>
 #include <common/type_to_string.h>
 #include <common/utils.h>
+#include <devtools/create-gossipstore.h>
 #include <fcntl.h>
 #include <gossipd/gen_gossip_store.h>
 #include <gossipd/gossip_store.h>
 #include <inttypes.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
 #include <wire/gen_peer_wire.h>
 
+
+
+struct scidsat * load_scid_file(FILE * scidfd) 
+{
+        int n;
+        fscanf(scidfd, "%d\n", &n);	
+	char title[16];
+        fscanf(scidfd, "%s\n", title);	
+	struct scidsat * scids = calloc(n, sizeof(scidsat));
```

Comment:
calloc() zeroes, which is unnecessary here.  Specifically, we use tal as an allocator, which provides a nice `scids = tal_arr(NULL, struct scidsat, n)`